module_1/src/GenreSelect/GenreSelect.js

```diff
@@ -0,0 +1,50 @@
+import './GenreSelect.css';
+import React, {useState} from 'react';
+import { IoMdArrowDropdown } from 'react-icons/io';
+
+
+function GenreSelect (props) {
+    const [isOpened, setOpened] = useState(true);
+    const [rotateIcon, setRotateIcon] = useState(false);
+    const [currentlySelectedArr, setCurrentlySelectedArr] = useState(props.currentlySelectedArr);
+    
+    const rotate = rotateIcon ? "rotate(180deg)" : "rotate(0)";
+    const iconStyles = { color: "#F65261", fontSize: "1.5em", position: 'absolute', top: '12px', right: '16px', transform: rotate  };
```

Comment:
you can manage this on css class level, the classnames library can make easier operation with them
https://www.npmjs.com/package/classnames